ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -241,6 +241,47 @@ tests:
       DISCONNECTED: "false"
       IP_STACK: v4
     workflow: baremetalds-assisted-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-multinode-spoke-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+    env:
+      ASSISTED_ZTP_CONFIG: |
+        SPOKE_CONTROLPLANE_AGENTS=3
+      DEVSCRIPTS_CONFIG: |
+        CLUSTER_NAME="ostest-master"
+        NUM_EXTRA_WORKERS=3
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=32768
+        EXTRA_WORKER_DISK=20
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
```

Comment:
We should be able to provide only the changed configuration without all the default
@osherdp IIRC we used the `conf` file in order to support it be possible.

Reply:
we just need to add/expand on what we already have, as we doing with the ``IP_STACK`` configuration
seems reasonable

Reply:
For the options here we have the ability to break out a lot of it with the same approach we did with `IP_STACK` or maybe moving it into its own workflow for multi-node. I wasn't sure which would be preferable. The only things that are the same here are `PROVISIONING_NETWORK_PROFILE` and `REDFISH_EMULATOR_IGNORE_BOOT_DEVICE`